tmt/steps/provision/minute.py

```diff
@@ -69,18 +70,55 @@ class ProvisionMinute(tmt.steps.provision.ProvisionPlugin):
             image: 1MT-Fedora-32
             flavor: m1.large
 
-    Available images and flavors can be listed using '1minutetip list'
-    and '1minutetip list-flavors'.
+    Available images and flavors can be listed using 'tmt run provision --how minute --list'
+    and 'tmt run provision --how minute --list-flavors'.
     """
 
     # Guest instance
     _guest = None
 
+    # API url for openstack gateway
+    api_url = ""
+
     # Supported methods
     _methods = [
         tmt.steps.Method(name='minute', doc=__doc__, order=50),
         ]
 
+    def _populate_api_url(self):
+        """ Get api url from 1minutetip script """
+        # Read API URL from 1minutetip script
+        try:
+            self.debug(f"Get the API URL from '{SCRIPT_PATH}'.")
+            script_content = self.read(SCRIPT_PATH)
+            match = re.search(API_URL_RE, script_content)
+            if not match:
+                raise tmt.utils.ProvisionError(
+                        f"Could not obtain API URL from '{SCRIPT_PATH}'.")
+            self.api_url = match.group('url')
+            self.debug('api_url', self.api_url, level=3)
+        except tmt.utils.FileError:
+            raise tmt.utils.ProvisionError(
+                f"File '{SCRIPT_PATH}' not found. Please install 1minutetip.")
+
+    def _filter_images_list_output(self, image_list_raw):
+        """ Prepare raw image list to terminal """
+        filter_out = ('new', 'obsolete', 'invalid')
+        image_list = list(filter(lambda line:
+            not line.endswith(filter_out) and line.startswith('1MT-'),
+            image_list_raw.splitlines()
+        ))
+        image_list.sort()
+
+        return image_list
+
+    def _print_images_list(self, image_list):
+        os_groups = ('Fedora', 'CentOS', 'RHEL-5' 'RHEL-6', 'RHEL-7', 'RHEL-ALT', 'RHEL-8', 'RHEL-9')
+        for os_group in os_groups:
```

Comment:
Can we have also holder for 'all other images'?

Reply:
What do you mean by all other images? This is minute plugin, which provides minute tip images... if you need other images like windows one, maybe some more common plugin for openstack should exists.

Reply:
Fair enough. We will just need to update minute.py once new group appears (centos-stream maybe?)

Reply:
Yes. In case of RHEL-(9 + 1) we should edit this.
Maybe we can add some sqlite (for example) database with groups internally and just read from it without code changes...